client/extensions/woocommerce/woocommerce-services/views/shipping-label/style.scss

```diff
@@ -1,103 +1,94 @@
 @import './label-purchase-modal/style';
-.shipping-label {
-	margin-bottom: 0;
-	text-align: center;
-
-	.shipping-label__payment .gridicon.notice__icon {
-		align-self: flex-start;
-		margin-top: 8px;
-	}
-
-	.shipping-label__item {
-		padding: 16px;
-		border-bottom: 1px solid #eee;
+.shipping-label__payment .gridicon.notice__icon {
+	align-self: flex-start;
+	margin-top: 8px;
+}
 
-		.shipping-label__item-created {
-			color: $gray;
-			font-size: 12px;
-		}
+.shipping-label__item {
+	// padding: 16px;
+	// border-bottom: 1px solid #eee;
 
-		&:last-child {
-			border: none;
-		}
+	// .shipping-label__item-created {
+	// 	color: $gray;
+	// 	font-size: 12px;
+	// }
 
-		p {
-			margin-bottom: 3px;
-			text-align: left;
+	// &:last-child {
+	// 	border: none;
+	// }
 
-			&.shipping-label__item-tracking {
-				margin-bottom: 16px;
-				font-size: 12px;
+	p {
+		margin-bottom: 3px;
+		text-align: left;
 
-				a:focus {
-					outline: none;
-					box-shadow: none;
-				}
-			}
+		&.shipping-label__item-tracking {
+			// margin-bottom: 16px;
+			font-size: 12px;
 
-			& .shipping-label__item-detail {
-				font-weight: bold;
-				font-size: 14px;
-				color: $gray-dark;
-				cursor: help;
-				text-decoration: underline;
-				text-decoration-color: $gray-dark;
-				text-decoration-style: dotted;
+			a:focus {
+				outline: none;
+				box-shadow: none;
 			}
 		}
 
-		.shipping-label__new-label-button {
-			width: 100%;
-			margin-top: 16px;
-		}
-
-		.shipping-label__item-actions {
-			display: flex;
-			justify-content: space-between;
-			font-size: 12px;
-			margin-bottom: 0;
+		// & .shipping-label__item-detail {
+		// 	font-weight: bold;
+		// 	font-size: 14px;
+		// 	color: $gray-dark;
+		// 	cursor: help;
+		// 	text-decoration: underline;
+		// 	text-decoration-color: $gray-dark;
+		// 	text-decoration-style: dotted;
+		// }
```

Comment:
There are a few places in this file with commented out code too, this should also be removed.